gradle/verification-metadata.xml

```diff
@@ -199,6 +200,7 @@
    <components>
       <component group="com.datastax.oss" name="java-driver-bom" version="4.14.1">
```

Comment:
Can we just add this to <trusted-keys>? It seems to be properly signed and has a signature 

https://repo1.maven.org/maven2/com/datastax/oss/java-driver-bom/4.14.1/

https://keyserver.ubuntu.com/pks/lookup?search=0x2DC48CBB4352B4953AF6F803D433B437192A0FD1&fingerprint=on&op=index